command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -22,8 +22,8 @@ def cleared(self, alarm):
     def bagger_dump(self):
         """Call online_bagger/dump service"""
         camera = '/camera/front/left/camera_info /camera/front/left/image_raw '
-        navigation = '/wrench /wrench_actual /c3_trajectory_generator/trajectory_v \
-                      /c3_trajectory_generator/waypoint /trajectory '
+        navigation = '/wrench /wrench_actual /c3_trajectory_generator/trajectory_v' + \
+                     '/c3_trajectory_generator/waypoint /trajectory /odom'
```

Comment:
I agree with this, it makes sense to have a central location for the bagging variables, even if that means creating a bag_kill variable and making it a combination of the ones we already have.